ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -40234,24 +40233,26 @@ periodics:
       secret:
         secretName: result-aggregator
 - agent: kubernetes
-  cluster: build04
-  cron: 0 0 * * 1
+  cluster: build05
+  cron: 0 12 1 1 *
   decorate: true
   decoration_config:
     skip_cloning: true
+    timeout: 5h0m0s
   extra_refs:
   - base_ref: master
     org: openshift
     repo: release
   labels:
-    ci-operator.openshift.io/cloud: packet-edge
-    ci-operator.openshift.io/cloud-cluster-profile: packet-assisted
+    ci-operator.openshift.io/cloud: equinix-ocp-metal
+    ci-operator.openshift.io/cloud-cluster-profile: equinix-ocp-metal
+    ci-operator.openshift.io/cluster: build05
     ci-operator.openshift.io/variant: nightly-4.12
     ci.openshift.io/generator: prowgen
     ci.openshift.io/no-builds: "true"
     job-release: "4.12"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ovn-assisted
+  name: periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-upgrade-ovn-ipv6
```

Comment:
You need to make sure that any periodic name changes, that may be referenced in [release configurations](https://github.com/openshift/release/tree/master/core-services/release-controller/_releases), are also updated to the new names.

Reply:
The two jobs I renamed arn't referenced in  release configurations, most of the patch/diff is the order of jobs getting moved around because of the rename of the compact jobs

Reply:
IIRC, there is a check that will catch it if you didn't rename in the release-config, so should be good.